extensions/amp-quicktext/0.1/test/validator-amp-quicktext.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
Please add the AMP Project license to this file, and update the test output to match.

Reply:
It looks like the license has been added, but the corresponding `.out` file does not show the license.

Reply:
Is this PR still outstanding?